### ISSUE TITLE
Inpurt script switches to json5 format

### ISSUE THREAD

**Issue by wanghan-iapcm** [deepmodeling/dpgen2#94]
The json5 format is a extension of the json format, and is more human readable and easier to write. The main advantages:
- support comments
- trailing commas allowed.
- no quotation needed for keys.

More details are found on [the official web site of json5](https://json5.org/)

The json5 is parsed by python package [pyjson5](https://pypi.org/project/json5/)

**Comment by link89** [comment]
I think YAML is good enough for most cases. If people feel that writing yaml/json directly is annoy, they could just write some Typescript script to generate YAML/JSON file. There are modern solutions like [cue-lang](https://cuelang.org/) to generate YAML configuration. Both Typescript and cue-lang are working well with VsCode if you define the type schema properly.

**Comment by njzjz** [comment]
> I think YAML is good enough for most cases.

I agree with it as YAML also supports all of these features (comments, no quotation, etc). However, it seems to me that dpgen2 has not supported YAML yet.
